snapshots/my_fs_backup_location/incompatible-snapshots

```diff
@@ -0,0 +1 @@
+{"incompatible-snapshots":[]}
```

Comment:
This comment is for the whole snapshots folder, since it is only used for testing, can you move it inside the tests folder